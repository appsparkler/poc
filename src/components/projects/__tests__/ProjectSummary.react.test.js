import React from 'react'
import ProjectSummary from '../ProjectSummary.react'
import renderer from 'react-test-renderer'

it('should render correctly', () => {
  expect.assertions(1)
  const props = {
    project: {
      id: '1',
      title: 'Hello World',
      content: 'This is project content written in plain text...',
    },
  }
  const component = renderer.create(<ProjectSummary {...props} />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
