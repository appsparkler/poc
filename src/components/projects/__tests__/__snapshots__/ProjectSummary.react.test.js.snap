// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="project-summary card"
>
  <div
    className="card-content grey-text text-darken-3"
  >
    <span
      className="card-title"
    >
      Hello World
    </span>
    <p>
      Posted By Net Ninja
    </p>
    <p
      className="grey-text"
    >
      3rd September, 2am
    </p>
  </div>
</div>
`;
